tools/ctc_segmentation/scripts/run_ctc_segmentation.py

```diff
@@ -116,10 +116,9 @@
 
         original_duration = len(signal) / sample_rate
         logging.debug(f'Duration: {original_duration}s, file_name: {path_audio}')
-        log_probs = asr_model.transcribe(paths2audio_files=[str(path_audio)], batch_size=1, logprobs=True)[0].cpu()
+        log_probs = asr_model.transcribe(paths2audio_files=[str(path_audio)], batch_size=1, logprobs=True)[0]
```

Comment:
Why revert the .cpu() here?

Reply:
Now the transcribe returns numpy array instead of tensors. So no need for call to cpu(). I asked @ekmb to check it already.